tests/integrations/linuxdeploy/test_LinuxDeploy__verify_plugins.py

```diff
@@ -222,8 +224,8 @@ def mock_downloads(url, download_path):
                 tmp_path
                 / "tools"
                 / "linuxdeploy_plugins"
-                / "sometool"
-                / "f3355f8e631ffc1abbb7afd37b36315f7846182ca2276c481fb9a43a7f4d239f"
+                / "network"
+                / "f93c7e6d04425e4ed7e533655d9898c4984f104711f4ffd3b3966cfc92500b2d"
```

Comment:
Since `linuxdeploy`.is_elf_file()` is actually trying to read this file, this needed to be updated to match the rest of the test; otherwise, the file wasn't found.